samples/siwx91x_ota/sample.yaml

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) 2025 Silicon Laboratories Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+sample:
+  name: HTTP/HTTPS OTAF
```

Comment:
TDH, this is the first time with this sample I hear about "OTAF". I seems the whole world uses "FOTA" acronym.